src/components/victory-axis/victory-axis.jsx

```diff
@@ -98,6 +99,20 @@ export default class VictoryAxis extends React.Component {
      */
     domain: CustomPropTypes.domain,
     /**
+     * The events prop attaches arbitrary event handlers to namespaced elements
+     * Parent events are only supported on standalone components i.e. top level svgs.
```

Comment:
Can you explain parent events a bit more? What are they used for, and what does the "standalone component" limitation mean to a user?